backend/app/tests/test_imap_client.py

```
@@ -307,6 +308,55 @@ def test_successful_connection(self):
+
+        assert success is True
+        assert stats["use_ssl"] is False
+        plain_imap.assert_called_once_with("protonmail-bridge", 143)
+        tls_imap.assert_not_called()
+
+    def test_starttls_imap_connection_upgrades_before_login(self):
+        client = IMAPClient(
+            server="imap.example.com",
+            port=143,
+            username="user",
+            password="password",
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to argument: "password"

(S106)